src/styles/quote.css

```diff
@@ -0,0 +1,84 @@
+.quotes-container {
+  position: relative;
+  transform: none;
+  left: 0;
+  max-width: 76em;
+  margin: auto;
+}
+
+@media only screen and (max-width: 720px) {
+  .quotes-container {
+      width: 45em;
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+      text-align: center;
+      justify-content: center;
+      position: relative;
+      margin-top: 190px;
+      margin-left: 165px;
+  }
+}
+
+@media only screen and (max-width: 485px) {
+  .quotes-container {
+      width: 45em;
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+      text-align: center;
+      justify-content: center;
+      position: relative;
+      margin-top: 130px;
+      margin-left: 50px;
+  }
+}
+
+.customer-blockquote {
+  padding: 24px;
+  width: calc(50% - 55px);
+  display: inline-block;
+  border-left: 2px solid rgba(0,0,0,0.05);
+  font-family: 'Poppins', sans-serif;
+  font-size: 1.267em;
+  color: #3d4351;
+  line-height: 1.6em;
+}
+
+.quotes-container > .customer-blockquote:last-child{
+  border-right: 2px solid rgba(0,0,0,0.05);
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```